nbconvert/preprocessors/execute.py

```diff
@@ -103,7 +103,18 @@ class ExecutePreprocessor(Preprocessor):
             """
             )
     ).tag(config=True)
-
+    
+    shutdown_kernel = Enum(['graceful', 'immediate', 'now'],
```

Comment:
I agree that it would be best to remove 'now'. Thanks so much for helping with this PR. It should be a helpful improvement.